wolfi-images/cloud-mi2.yaml

```
@@ -0,0 +1,21 @@
+include: ./sourcegraph-base.yaml
+
+contents:
+  packages:
+    # Included by existing SG base image
+    - tini
+    - mailcap
+    - terraform=1.3.9-r2
```

```suggestion

    ## mi2 packages
    - terraform=1.3.9-r2
```

Added a comment above mi2-specific packages, just for consistency with other base images